study.md

```diff
@@ -31,7 +31,10 @@ In your own words, define what the responsibilities of the model layer are in
 Rails.
 
 ```md
-<!-- your answer here -->
+The model layer is the one who actually does the work.  It is a class that
+handles the dirty work of retrieving and storing data from the database. The
+model also inherits the ActiveRecord class that is actually quite powerful.
+It can recognize anonymous functions and match it to an equivalent.
```

Comment:
I'm not sure what this means.

What does it mean for it recognize anonymous functions? Are there anonymous functions in ruby? What are they called, and how are they used by the model layer?